cmd/helm/install.go

```diff
@@ -28,38 +35,53 @@ var installCmd = &cobra.Command{
 }
 
 func runInstall(cmd *cobra.Command, args []string) error {
-	if len(args) == 0 {
-		return fmt.Errorf("This command needs at least one argument, the name of the chart.")
-	}
-
-	ch, err := loadChart(args[0])
-	if err != nil {
-		return err
-	}
+	validateInstallEnv(args)
 
-	res, err := helm.InstallRelease(ch)
+	res, err := helm.InstallRelease(installArg)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("release.name:   %s\n", res.Release.Name)
-	fmt.Printf("release.chart:  %s\n", res.Release.Chart.Metadata.Name)
-	fmt.Printf("release.status: %s\n", res.Release.Info.Status.Code)
+	printRelease(res.GetRelease())
 
 	return nil
 }
 
-func loadChart(path string) (*chart.Chart, error) {
-	path, err := filepath.Abs(path)
-	if err != nil {
-		return nil, err
+// TODO -- Display formatted description of install release status / info.
+// 		   Might be friendly to wrap our proto model with pretty-printers.
+//
+func printRelease(rel *release.Release) {
+	if verbose {
+		if rel != nil {
+			fmt.Printf("release.name:   %s\n", rel.Name)
+			fmt.Printf("release.info:   %s\n", rel.GetInfo())
+			fmt.Printf("release.chart:  %s\n", rel.GetChart())
+		}
 	}
+}
 
-	if fi, err := os.Stat(path); err != nil {
-		return nil, err
-	} else if fi.IsDir() {
-		return chart.LoadDir(path)
+func validateInstallEnv(args []string) {
+	if len(args) > 0 {
+		installArg = args[0]
+	} else {
+		fatalf("This command needs at least one argument, the name of the chart.")
 	}
 
-	return chart.Load(path)
+	if tillerHost == "" {
+		if tillerHost = os.Getenv(hostEnvVar); tillerHost == "" {
+			fatalf("Missing tiller address")
```

Comment:
Why not use a default local address that can be overridden with a flag or envar?